RMActionController/RMActionController.m

```diff
@@ -293,12 +293,16 @@ - (void)setupTopContainerElements {
     if(!self.disableBlurEffects) {
         self.topContainer.backgroundColor = [UIColor clearColor];
     } else {
-        self.topContainer.backgroundColor = [UIColor whiteColor];
+        if (@available(iOS 13.0, *)) {
+            self.topContainer.backgroundColor = [UIColor secondarySystemBackgroundColor];
+        }else {
+            self.topContainer.backgroundColor = [UIColor whiteColor];
+        }
     }
 
     //Debugging Accessibility Labels
 #ifdef DEBUG
-    self.topContainer.accessibilityLabel = @"TopContainer";
+    self.topContainer.accessibilityLabel = @"TopContainer   ";
```

Comment:
Is this change intended?

Reply:
Apologies, I just pushed commit acbec02 to fix this typo